test/unit/test_db_helpers.py

```diff
@@ -43,7 +46,8 @@ def test_oracle_connect_exceptions():
      'DRIVER=test driver;SERVER=tcp:server;PORT=1521;DATABASE=testdb;UID=testuser;PWD=mypassword'), # NOQA
     (POSTGRESDB, psycopg2,
      'host=server port=1521 dbname=testdb user=testuser password=mypassword'),
-    (SQLITEDB, sqlite3, '/myfile.db')
+    (SQLITEDB, sqlite3, '/myfile.db'),
+    (INFORMIXDB, ibm_db_dbi, 'database=testDb;hostname=localhost;port=1111;protocol=tcpip;uid=user;pwd=mypassword')
```

Comment:
Line length.  See `MSSQLDB` example above for line splitting and turning off Flake8 check for line.